topics/fair/tutorials/data-management-peatlands/tutorial.md

```diff
@@ -0,0 +1,113 @@
+
+tags:
+- fair
+- open
+- data management plan
+priority: 1
+contributions:
+  authorship:
+    - GabrielaLopez
+    - kkamieniecka
+    - robertmand
+    - angiolini
+    - poterlowicz-lab
+  editing:
+    - hexylena
+subtopic: fair-data
+
+---
+
+
+Peatland monitoring and research generates data that is used for management, conservation and policy related to these ecosystems. To ensure that your project data is collected systematically and can be analysed, shared, and re-used it is important to have a data management plan.
+
+This tutorial is a short introduction to writing a Data Management Plan (DMP) with examples for peatland monitoring projects and initiatives.
+
+You can find more information on DMPs in the [Research Data Management Toolkit for Life Sciences]( https://rdmkit.elixir-europe.org/)
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# What is a Data Management Plan?
+
+A Data Management Plan (DMP) is a live document that provides a framework for managing data material during and after the research project.
+
+A data management plan will facilitate the process of collecting and managing your peatland data and will allow you to discuss with your team how to share data, considering the requirements of your organisation and funders.
+
+# Key components of a DMP
+DMP templates vary by institution and funding agencies. 
+
+| DMP component  | What to include|
+| ------------- | ------------- |
+| Descriptions of the data  |    &bull; Project details <br> &bull; Types of data <br> collected and generated <br> &bull; Origin of data <br> &bull; Format and size of data |
+|Data collection/generation | &bull; Data collection methods <br> &bull; Data quality standards| 
+|Data management, documentation, and curation | &bull; Storage and accessibility <br> &bull; Metadata standards and documentation <br> &bull; Curation and preservation |  
+|Data sharing and access | &bull; Where the data will be shared <br> &bull; When will the data be available |
+|Data security (where relevant)|&bull; Risks to data security<br> &bull; GDPR considerations|
+|Capabilities| &bull; Evaluation of institutional capabilities to preserve and manage data|
+|Maintaining and implementing the Data Management Plan|&bull; Actions to maintain and deliver the DMP |
+|Environmental considerations|&bull; Energy cost considerations related to data management|
+|Responsibilities|&bull; Agreement on the responsibilities of named individuals for data management|
+|Relevant institutional, departmental or study policies on data sharing and data security| &bull; Review of institutional policies for data management|
+
+## Hands-on: Create a DMP for peatland research
+* Download the DMP template for [Peatland Research and Monitoring](https://zenodo.org/records/11185774).
+* Fill in the DMP template using the prompt questions.
+* Review your DMP with your team and collaborators.
```

Comment:
Would it be possible to include a completed exemplar here?

Reply:
Hi Helena, sorry I've missed your comment. We will add an example and attach supplementary video after workshop on Thursday.

Reply:
@hexylena @shiltemann Hi both, sorry to bother you can we merge it this way now so we can update after workshop and still distribute the lesson URL? Thank you!

Reply:
Hi @hexylena we have now example document on zenodo and video of the interactive training session, please merge

Reply:
hi @kkamieniecka apologies for missing the jun6 message. Yes of course!